src/enums/zones.ts

```diff
@@ -2,14 +2,20 @@ import { COLOR_MAP } from './interventions';
 import { fail } from 'assert';
 
 export enum ChartType {
-  CASE_GROWTH_RATE = 'Infection rate',
-  POSITIVE_TESTS = 'Positive tests',
+  CASE_GROWTH_RATE = 'Infection growth rate',
+  POSITIVE_TESTS = 'Positive test rate',
   HOSPITAL_USAGE = 'COVID ICU usage',
 }
 
+export const ChartTypeToMetricName = {
+  [ChartType.CASE_GROWTH_RATE]: 'Infection growth rate',
+  [ChartType.POSITIVE_TESTS]: 'Positive test rate',
+  [ChartType.HOSPITAL_USAGE]: 'COVID ICU usage',
+};
```

Comment:
Maybe I'm blind, but how is this different than the ChartTypeToTitle mapping?  Are we expecting them to diverge?  Or if this is a "we should clean this up" thing, can you add a TODO?